Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -460,6 +461,7 @@ class SpringAnimation extends Animation {
     this._initialVelocity = config.velocity;
     this._lastVelocity = withDefault(config.velocity, 0);
     this._toValue = config.toValue;
```

Comment:
I believe @eslint-bot may need an update. I'm not getting this warning when running eslint locally. Also `useNativeDriver` field is declared in `AnimationConfig` type that is a parent type of `SpringAnimationConfigSingle`